tests/baselines/reference/recursiveBaseConstructorCreation3.errors.txt

```diff
@@ -14,6 +14,6 @@ tests/cases/compiler/recursiveBaseConstructorCreation3.ts(10,18): error TS2339:
     }
     
     var bar = new xyz(); // Error: Invalid 'new' expression.

```

Comment:
I can get back to the old behaviour without too much trouble. I'll see if I can also get rid of the "Property 'foo' does not exist on type 'xyz'" as well.